query-engine/connector-test-kit-rs/qe-setup/src/cockroachdb.rs

```diff
@@ -1,6 +1,6 @@
-use migration_core::migration_connector::{ConnectorError, ConnectorResult};
 use once_cell::sync::OnceCell;
 use quaint::{prelude::*, single::Quaint};
+use schema_core::schema_connector::{ConnectorError, ConnectorResult};
```

Comment:
Is this something that we want to update the naming convention of, wrt `migration_connector` -> `schema_connector` or are these used for introspection too?

Reply:
The connector is also used for introspection.